lib/drivers/medtronic/medtronicSimulator.js

```diff
@@ -83,6 +83,17 @@ exports.make = function(config){
     events.push(e);
   }
 
+  function addDurationToTime(time, timezoneOffset, duration) {
```

Comment:
I'd consider changing the function name and first param to `deviceTime` for clarity, but up to you...